admin/javascript/src/config.js

```diff
@@ -26,8 +26,9 @@ class Config {
     const topLevelRoutes = [];
 
     Object.keys(window.ss.config.sections).forEach((key) => {
+      // Top level routes should have only one url-segment beneath admin/ and have a trailing slash
       const route = window.ss.config.sections[key].route;
-      const isTopLevelRoute = route.indexOf('/') === -1;
+      const isTopLevelRoute = route.match(/^admin\/([^\/]+)\/$/);
```

Comment:
Do these routes always end in `/`? Feels a bit brittle, maybe it needs a `?` before `$`?